src/main/java/common/Winner.java

```diff
@@ -0,0 +1,20 @@
+package common;
```

Comment:
Can I suggest renaming this package to be more specific? Perhaps `enumeration`? The danger of a `common` package is it becomes a 'dumping ground' for non-specific code